android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -62,14 +62,15 @@ public void startWithOptions(final PluginCall capOptions) {
         SentryAndroid.init(
             this.getContext(),
             options -> {
+                if (capOptions.getData().has("debug") && capOptions.getBoolean("debug")) {
+                    options.setDebug(true);
+                    logger.setLevel(Level.INFO);
```

Comment:
We usually have `DEBUG` by default. User can lower it if it's too noisy, but out of the box we give as much information as we can

Reply:
logger.setLevel(Level.INFO); is only within the `SentryCapacitor.java`, its not the Android SDK logger, so its fine